diff --git a/dist/cjs/index.js b/dist/cjs/index.js
index b4242b0b847d33d8544832bfb2d8723a884a3072..6f96daa9560df9b35ada38f86f561187e4c932bf 100644
--- a/dist/cjs/index.js
+++ b/dist/cjs/index.js	
@@ -185,7 +185,7 @@ var _PopupView = class _PopupView extends import_node_events.EventEmitter {
       roundedCorners: false,
       webPreferences: {
         session: opts.session,
-        sandbox: true,
+        sandbox: false,
         nodeIntegration: false,
         nodeIntegrationInWorker: false,
         contextIsolation: true,
